product_secondary_unit/models/product_secondary_unit_mixin.py

```diff
@@ -131,3 +130,12 @@ def _onchange_helper_product_uom_for_secondary(self):
             precision_rounding=self.secondary_uom_id.uom_id.rounding,
         )
         self.secondary_uom_qty = qty
+
+    @api.model
+    def default_get(self, fields_list):
+        defaults = super().default_get(fields_list)
+        if self.secondary_uom_id and not self.env.context.get(
```

Comment:
Is it necessary to check `self.secondary_uom_id`?

Reply:
I ithink yes... I do not set a default value to 1.0 if not secondary uom set...